packages/workbox-webpack-plugin/src/lib/utils/webpack-asset.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Creates a webpack asset from a string that can be added to a compilation
```

Comment:
Not sure what this does or why it's needed, could you add some more context? Sorry I'm such a webpack noob :(

Reply:
I understand the reason this is needed, but agree for future contributors it would be useful to expand on why this is needed.

Reply:
I will elaborate what this is in the docs.

These are the minimum properties that webpack expects a single asset to have. Given a string (`asset`) we can structure it for webpack without any other arguments.